websites/R/Roblox/dist/metadata.json

```diff
@@ -9,35 +9,50 @@
 			"name": "YJJcoolcool",
 			"id": "300929818231308288"
 		},
+		{
+			"name": "Vex",
+			"id": "702007645976133693"
+		},
 		{
 			"name": "Kyrie",
 			"id": "368399721494216706"
 		}
 	],
 	"service": "Roblox",
 	"description": {
-		"en": "Roblox's mission is to bring the world together through play.",
+		"en": "Roblox’s mission is to bring the world together through play.",
 		"ro": "Misiunea Roblox-ului este de a aduce lumea mai aproape să se joace împreună.",
+		"es": "La misión de Roblox es unir al mundo por medio del juego.",
 		"nl": "De missie van Roblox is om de wereld samen te brengen door te spelen.",
-		"zh_CN": "Roblox的使命是通过游戏将世界融合在一起。",
-		"vi_VN": "Nhiệm vụ của Roblox là đem thế giới lại gần nhau qua trò chơi."
```

Comment:
Why was the `vi_VN` description deleted?

Reply:
I didn't do it on purpose, my metadata file wasn't up-to-date. Thanks for pointing it out